### ISSUE TITLE
[DatePicker] [[DateRangePicker]]: On accessing date pickers through screen reader, corresponding ‘Day’ and ‘week count’ information is not read

### ISSUE THREAD

**Issue by I506645** [SAP/ui5-webcomponents#9441]
### Describe the bug

On accessing date pickers through screen reader, corresponding ‘Day’ and ‘week count’ information is not read. 

Example: On accessing Date: ‘19’, Month: ‘April’ and Year: ‘2019’ within the date picker, screen reader reads it as ‘April 19, 2019’.

### Reproduction steps

Access the DatePicker or DateRangePicker component using Screen-reader.

### Expected Behaviour

Screen reader should read all the information available on the screen.

On accessing all the dates with the date picker, corresponding ‘Day’ and ‘week count’ information should also be read.

Example: On accessing Date: ‘19’, Month: ‘April’ and Year: ‘2019’ within the date picker, it should be read as ‘Week 16, Friday, April 19, 2019’.

### UI5 Web Components for React Version

1.17.3

### UI5 Web Components Version

1.17.0

### Browser

Chrome

### Operating System

Windows

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by GerganaKremenska** [comment]
Hello @SAP/ui5-webcomponents-topic-b ,

Please review the issue, that not enough information is read out on the DatePicker - https://sap.github.io/ui5-webcomponents/nightly/components/DatePicker/

Note: I have checked the best [practice example ](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepicker-dialog/) for date picker, reads out only the date month and year regarding date information.

Please take over!

Best Regards,
Gergana

**Comment by unazko** [comment]
Hello @I506645, @GerganaKremenska,

The described behavior is an already known JAWS issue under number 0093 in the wiki.
Not sure you have access to the wiki so the following is described there:

> sap.ui.unified.Calendar inside a dialog container:
> 
> - The calendar role description isn’t announced
> - The referred description isn’t read out
> - The counting information from the grid role isn’t read out
> - No week information and day name gets announced with Jaws and VPC on 
> 
> Remove the “dialog” role to see the expected announcements.
> 
> With NVDA everything is announced with the “dialog” role.

The issue is also reproducible with native HTML via the following sample:
https://jsfiddle.net/unazko/qaodgtes/8/

I'm closing the incident as there is nothing we could do about this JAWS issue.

Best regards,
Boyan
